org.rolisteam.rolisteam.json

```diff
@@ -0,0 +1,90 @@
+{
+  "id": "org.rolisteam.rolisteam",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.12",
```

Comment:
Since you opened this PR, 5.15 has been released. Mind giving it a try?

Reply:
I believe the 5.15 of that framework will bring Qt5.15 and this version of Rolisteam is not compatible with it. Or at least, I'm doing of the testing under Qt5.12. The next release will support Qt5.15. I'm not really fan about that even it should compile.

Reply:
Yeah, that's fair. Let's keep it as is then.